### ISSUE TITLE
Scrolling on iOS

### ISSUE THREAD

**Issue by Nattefrost** [quark-dev/Phonon-Framework#144]
Hi.
While our app (and phonon) works flawlessly on android (thanks for that!), we recently found that scrolling is somewhat buggy on iOS. 

When scrolling towards the bottom of a scrollable element, it look like the content is going out the window, leaving a big empty area at the bottom. Then scrolling back up is messy, you have to go really slowly. 

We witnessed the same behaviour when trying the phonon kitchen sink (tables section) on iOS (Iphone SE, IOS 9.3.2 - Chrome 50.0.2661.95 & Safari 601.1)

So is it a thing the phonon devs are aware of (haven't found another ticket with this exact problem), is it related to phonon or to iOS ? If it is the former do you aim to fix it?
Hope you guys can help =)

**Comment by Qathom** [comment]
Hi @Nattefrost,

Thank you for reporting this.
I am not aware of this kind of bug sadly.

Can you please give me an example of your content?
Are you using Apache Cordova with Phonon?

Thanks in advance.

**Comment by Nattefrost** [comment]
Ok we got a few screenshots, some things have been hidden but it should not be a problem for you to see the strange behaviour.
http://imgbox.com/YU9IpjZL
http://imgbox.com/M1MXuPr6
http://imgbox.com/4n9zqooI

**Comment by Nattefrost** [comment]
Have you had time to investigate the matter ? On our side we haven't found anything new since our last post except the fact that we have this behaviour from iOS6 and onwards (we don't know for versions before iOS6). 
On the other hand I started fiddling with cordova but not enough to supply any useful information.
Anyone got an idea of what could be going on based on the screenshots I provided ?

**Comment by Qathom** [comment]
Hi @Nattefrost,

I don't think that Cordova will make a difference about scrolling.
There is a problem of height, padding or margin with `.tab-content` or the main `.content` selectors.
Sadly, I don't have iOS 6-7 to debug it..
Is there a difference if you remove ```.padded-full```?

```
    <div class="content" data-tab-contents="true" data-disable-swipe="false" data-tab-default="1" >
        <div class="tab-content">
            <div class="padded-full">
                <h1>Tab 1</h1>
                <p>With padding</p>
            </div>
        </div>
        <div class="tab-content">
            <div>
                <h1>Tab 2</h1>
                <p>No padding</p>
            </div>
        </div>
    </div>
```

**Comment by Nattefrost** [comment]
@Qathom We tried your solution, no results.
But I somehow resolved the problem and I think it had to do with the phonon.dialogs. They were not visible but still in the html page and it disrupted scrolling, I added display: none when closing them and it seems to work now. Strange that it caused this on iOS only though.
 We have other problems related to iOS but I'll post when -and if- I'm sure it has to do with Phonon and not the way iOS handles some stuff.

As far as i'm concerned this issue is closed. Thanks for your time, still.
